test/suite/test_rollback_to_stable10.py

```diff
@@ -178,7 +178,9 @@ def test_rollback_to_stable(self):
         self.assertGreaterEqual(upd_aborted, 0)
         self.assertGreater(pages_visited, 0)
         self.assertGreaterEqual(hs_removed, 0)
-        self.assertGreater(hs_sweep, 0)
+        # Eviction won't see any of the newer updates that happen concurrently
```

Comment:
Are these comments related to the change?

Reply:
Yeah, I believe `hs_sweep` corresponds to updates removed from history store as a result of rollback to stable. So since eviction doesn't see those newer updates, there's nothing in the history store that needs to get removed when we perform rollback to stable.

I'll think about how to reword this.